_opensearch/rest-api/create-alias.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Create alias
+parent: REST API reference
+grand_parent: OpenSearch
+nav_order: 8
+---
+
+# Create alias
+
+An alias is a virtual pointer that you can use to reference one or more indices. For example, if you have continuous streams of incoming data but only want to search through the past seven days at any time, you can use an alias to point to only indices created within the past seven days.
+
+## Example
+
+```json
+POST _aliases
+{
+  "actions": [
+    {
+      "add": {
+        "index": "movies",
+        "alias": "movies-alias1"
+      }
+    }
+  ]
+}
+```
+
+## URL parameters
+
+Both create alias parameters are optional.
```

Comment:
I'd say "All alias parameters are optional." since a) this isn't just alias creation and b) that way we don't have to change this sentence if we ever add a third parameter.